csvcubed/csvcubed/cli/entrypoint.py

```diff
@@ -0,0 +1,41 @@
+"""
+CLI
+---
+
+The *Command Line Interface* for `csvcubed inspect`.
+"""
+
+
+import click
+import colorama
```

Comment:
@canwaf

Reply:
@robons @canwaf I am not using `colorama`. Just imported it and initialised it in the `entry_point()`. I can remove it if you want.

Reply:
Yes, please.

Reply:
Removed colorama.